modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestNode.java

```diff
@@ -81,7 +80,7 @@
 public class TestNode implements LifecycleAware {
     private final String nodeName;
     private final SchemaPlus schema;
-    private final PrepareService prepareService;
+    private final PrepareServiceImpl prepareService;
```

Comment:
TestNode must use interface, not implementation

Reply:
fixed

Reply:
well, no. Now it's even worse: interface PrepareService has two similar method, but one of them uses cache while other doesn't. If you don't believe in 0-sized caffeine cache, then introduce an abstraction for cache and provide different implementations in tests and production configuration